lib/usd/ui/layerEditor/mayaLayerEditorWindow.cpp

```diff
@@ -207,6 +208,16 @@ void MayaLayerEditorWindow::lockLayer()
     }
 }
 
+void MayaLayerEditorWindow::lockLayerAndSubLayers()
+{
+    auto item = treeView()->currentLayerItem();
+    if (item != nullptr) {
+        QString name = item->isLocked() ? "Unlock Layer and Sublayers" : "Lock Layer and Sublayers";
```

Comment:
I had a quick look and it seems like this string is used for the undo name. Is that right? Is there a reason then you didn't also put these strings in the stringResources.h?

Reply:
This seems to be the pattern for the menu items in this file. I've made a EMSUSD-1096 to address all the strings in this file.

Reply:
Okay thanks for checking.